src/main/java/de/theit/jenkins/crowd/CrowdConfigurationService.java

```diff
@@ -886,7 +895,7 @@ private <T> void setValueToCache(String key, T value, CacheMap<String, T> cacheO
 
         synchronized (this) {
             if (cacheObj == null) {
-                cacheObj = new CacheMap<>(cacheSize);
+                return;
```

Comment:
this line dont have to be in sync right now but it's ok.
I will move this later on.